test-jw/src/App.js

```diff
@@ -0,0 +1,11 @@
+import { VideoPlayer } from './bundle.esm'
```

Comment:
This is loading a different bundle.esm than the one being symlinked.

Reply:
The symlink command is `ln -s dist/bundle.esm.js test-jw/ ` which maps to `./bundle.esm` don't need to add the .js extension. Am I missing something else?